otx/algorithms/detection/adapters/mmdet/task.py

```diff
@@ -396,8 +396,6 @@ def hook(module, inp, outp):
             saliency_hook: Union[nullcontext, BaseRecordingForwardHook] = nullcontext()
         else:
             raw_model = feature_model
-            if raw_model.__class__.__name__ == "NNCFNetwork":
-                raw_model = raw_model.get_nncf_wrapped_model()
```

Comment:
Could you explain why we can remove this line?

Reply:
That happened because of changes in NNCF. They improved the process of wrapping, so get_nncf_wrapped_model() method is no longer required:
https://github.com/openvinotoolkit/nncf/blob/develop/nncf/torch/nncf_network.py#LL980C11-L980C11